src/Microsoft.IdentityModel.Clients.ActiveDirectory/Internal/ClientCreds/ClientKey.cs

```diff
@@ -43,6 +43,7 @@ public ClientKey(string clientId)
 
             this.ClientId = clientId;
             this.HasCredential = false;
+            this.SendX5c = false;
```

Comment:
Why don't aren't these constructors calling each other using : this(....)?

Reply:
@travis.walker@microsoft.com you can simplify by making the default constructor private and setting the 2 values which are common for all constructors. on top the new overload which sets the sendX5c can as Brent mentions be called using this as well. You can also consider removing the one with-out the bool param forcing it to be specified? 

---
In reply to: [164022484](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/959#discussion_r164022484) [](ancestors = 164022484)